learn/advanced/faceted_search.md

```diff
@@ -0,0 +1,159 @@
+
+```json
+{
+  "hits":[
+    …
+  ],
+  "query":"pie server",
+  "processingTimeMs":9,
+  "limit":20,
+  "offset":0,
+  "estimatedTotalHits":25,
+  "facetDistribution":{
+    "brand":{
+      "All Things Rae Dunn":1,
+      "BEERBONG .COM":1,
+      "Dr. Pet":1,
+      …
+      "senover":1
+    },
+    "rating":{
+      "0":7,
+      …
+      "4.8":1,
+      "5":3
+    },
+  },
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+}
+```
+
+### Facet distribution
+
+The `facetDistribution` object contains the number of matching documents distributed among the values of a given facet. The `facets` search parameter expects an array of strings. Each string is an attribute present in the `filterableAttributes` list.
+
+The following response shows the facet distribution when searching for `pie server`:
+
+```json
+{
+  …
+  "facetDistribution":{
+    "brand":{
+      "All Things Rae Dunn":1,
+      "BEERBONG .COM":1,
+      "Dr. Pet":1,
+      …
+      "senover":1
+    },
+    "rating":{
+      "0":7,
+      …
+      "4.8":1,
+      "5":3
+    }
+  }
+  …
+}
+```
+
+`facetDistribution` contains an object for every given facet. For each of these facets, there is another object containing all the different values and the count of matching documents. Note that zero values will not be returned: if there are no `pie server` products by a certain brand, it is not displayed.
+
+::: note
+By default, `facets` returns a maximum of 100 facet values for each faceted field. You can change this value using the `maxValuesPerFacet` property of the [`faceting` index settings](/reference/api/settings.md#faceting).
+:::
+
+### Facet stats
+
+When using the `facets` parameter, any matching documents with facets containing numeric values are displayed in a `facetStats` object. `facetStats` contains the numeric minimum (`min`) and maximum (`max`) values per facet for all documents matching the search query.
+
+If none of the matching documents have a numeric value for a facet, that facet is not included in the `facetStats` object.
+
+Meilisearch ignores string values like `"21"` when computing `facetStats`.
+
+The following response shows the `facetStats` when searching for `pie server`:
+
+```json
+{
+  …
+  "facetStats":{
+    "rating":{
+      "min":0.0,
+      "max":5.0
+    }
+  }
+  …
+}  
+```
+
+Since `rating` was the only numeric facet in our example, it is returned in the `facetStats` object.
+
+## Facet types
+
+### Single select facets
+
+Single-select facets allow the selection of a single value per facet. Using our `ecommerce` index, this means if you have a facet for `brand` with the values `BrandA`, `BrandB`, and `BrandC`. Single select facets will allow you to view products from one brand at a time. If you want to view `BrandA` and `BrandB` products, you will have to try both searches individually. If you want to further narrow down results using the `reviews` facet, you would need to do so separately for both brands.
```

Comment:
I think we need to be a bit more didactic with this. First paragraph should be a clear sentence defining what single-select facets are in Meilisearch.

Second paragraph explains the use-case: "single-select facets are helpful when [something]".

Third paragraph is a bit longer and illustrates the use-case with our simplified sample dataset: "with single-select facets, clicking on 'Nike' and 'sportswear' will only show sportswear products with a 'Nike' brand." 

A video/gif illustrating this would be super helpful.

Reply:
Oh, one last thing: explain what users need to do to use single-select facets with Meilisearch. Is it simply a matter of chaining filter with `AND` operators? Is it something else? Whatever it is, the user needs to know. (This also counts for multi-select facets)